### PR TITLE
Added tests for parser preferred time zone handling #3287

### PR DESCRIPTION
**Pull request by joachimmetz**
Added tests for parser preferred time zone handling #3287

### CODE REVIEW

File: tests/parsers/xchatlog.py [hunk 1]
```diff
@@ -14,9 +14,25 @@
 class XChatLogUnitTest(test_lib.ParserTestCase):
   """Tests for the xchatlog parser."""
 
-  def testParse(self):
+  def testParseWith(self):
```
Comment (rgayon) [inline tests/parsers/xchatlog.py:17]: *testParseWithTimeZone

**Review by rgayon — APPROVED** [review]
Just 1 nit

File: tests/parsers/xchatlog.py (hunk 1 above)
Comment (joachimmetz) [reply tests/parsers/xchatlog.py:17]: good catch

Comment (joachimmetz) [reply tests/parsers/xchatlog.py:17]: this should be just testParse

**Pull request merged**